### DISCUSSION TITLE
Unable to whitelist Chrome Extension in the dashboard to use Web3Auth from inside it

### DISCUSSION THREAD

**Discussion by mrobokit** [Web3Auth/Web3Auth#690] [Help]
I am bootstrapping the simplest create-react-app chrome extension and i want to be able to use Web3Auth from within the extension. When i inspect the popup html page, this is the error i get:

![image](https://user-images.githubusercontent.com/20832842/194417005-3be46f93-faee-45af-a7a3-5b58e1b87451.png)

I have opened the Dashboard and whitelisted as the error message suggest, however it doesn't seem to have any effect.
Could you please let us know if it's possible to effectively run Web3Auth from within a Chrome Extension like shown in the screenshot? If yes, then how?

**Comment by shahbaz17** [comment]
Hey @manzuni 

Sorry for the delay in response. We have a chrome extension example in [our examples repo](https://github.com/Web3Auth/examples/tree/main/web-core-sdk/browser-extensions/chrome-extension-web3auth-example). You can try this out.

- Add `chrome-extension://EXTENSION_ID` to [Web3Auth Dashboard](https://dashboard.web3auth.io)'s **Whitelist URLs**.
